thinknimble/tn-spa-bootstrapper
# Feature/Vue3 Tailwind setup + login/signup page styling

## What this does

Configure [TailwindCSS](https://tailwindcss.com/) for Vue3 projects created with the bootstrapper to be utilized from initial project creation.

Update the styling for the `Login` and SignUp` pages to be a bit more standardized to most of the project stylings we see.

## Checklist
- [x] Add `tailwindcss` npm package
- [x] Add `postcss` npm package
- [x] Add `autoprefixer` npm package
- [x] Add `tailwind.config.js` file and update as needed w/ desired paths
- [x] Add `postcss.config.js` file
- [x] Add `main.css` file w/ Tailwind imports
- [x] Import `main.css` into `main.js`
- [x] Style `Login` view and associated components
- [x] Style `SignUp` view and associate components


## How to test

Step 1: Open the review app and navigate through the existing pages to view the updating styling created with TailwindCSS.

Step 2:  Create/run a bootstrapper app with this branch (`feature/vue-tailwind-setup`) and update the Tailwind stylings/classes as desired to see different results!
